### PR TITLE
DUOS-1254[risk=no] Initial DAR Collection Review structure with header sub-component

### PR DESCRIPTION
**Pull request by JVThomas**
Partially addresses [DUOS-1254](https://broadworkbench.atlassian.net/browse/DUOS-1254)

PR creates initial structure for the `DarCollectionReview` component along with sub-components for the header/skeleton loaders. PR also includes update to `TabControl` component to render out skeleton loaders if a loading variable is provided to props.

Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/components/TabControl.js [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
   return labels.map((label) =>
-    h(SelectableText, {
-      label,
-      key: `${label}-button`,
-      fontSize: '1.8rem',
-      componentType: label,
-      setSelected: setSelectedTab,
-      selectedType: selectedTab
-    })
+    !isLoading ?
```
Comment (quazi-h) [inline src/components/TabControl.js:6]: Was the previous behavior here to show the SelectableText by default? With the new changes, if isLoading is false would we still show SelectableText?

**Review by quazi-h — APPROVED** [review]
The new DarCollectionReview and header components look good to me.

**Review by rushtong — APPROVED** [review]
Looks good on screen - no real feedback on anything at this point 👍🏽

File: src/components/TabControl.js (hunk 1 above)
Comment (JVThomas) [reply src/components/TabControl.js:6]: Yeah the initial setup would always show SelectableText. The idea now is to show SelectableText if `isLoading` is true (comes from the parent controller). If it's not, it'll render the second div, which shows a `div` with an animated gradient (a simple loading effect). If the parent finishes loading, it should update its `isLoading` value to trigger downstream changes on this component.

As a safety measure if `isLoading` doesn't have a value the component will default it as false to ensure that `SelectableText` is rendered.

**Comment by JVThomas** [comment]
setTimeout warning on Codacy is a false positive

**Pull request merged**